java/client/src/org/openqa/selenium/safari/SafariDriver.java

```diff
@@ -37,20 +47,15 @@
  */
 public class SafariDriver extends RemoteWebDriver {
 
-  /**
-   * Capability to force usage of the deprecated SafariDriver extension while running
-   * on macOS Sierra.
-   *
-   * <pre>
-   *   DesiredCapabilities safariCap = DesiredCapabilities.Safari();
-   *   safariCap.setCapability(SafariDriver.USE_LEGACY_DRIVER_CAPABILITY, true);
-   *   WebDriver driver = new SafariDriver(safariCap);
-   * </pre>
-   */
-  public static final String USE_LEGACY_DRIVER_CAPABILITY = "useLegacyDriver";
-
   private SafariDriverService service;
 
+  // Legacy Window API
+  protected final static String SET_WINDOW_SIZE = "setWindowSize";
```

Comment:
Since it's a constant, we may even make it public, like the original constants in the DriverCommand interface.